runtime/doc/msgpack_rpc.txt

```diff
@@ -270,10 +270,15 @@ a dictionary with these (optional) keys:
 				Defaults to false.
 
 Nvim will then send msgpack-rpc notifications, with the method name "redraw"
-and a single argument, an array of screen updates (described below).
-These should be processed in order. Preferably the user should only be able to
-see the screen state after all updates are processed (not any intermediate
-state after processing only a part of the array).
+and a single argument, an array of screen updates (described below).  These
+should be processed in order. Preferably the user should only be able to see
+the screen state after all updates in the same "redraw" event are processed
+(not any intermediate state after processing only a part of the array).
+
+New update kinds will be added in later versions of Nvim.  It is safe for a
+client to ignore new updates it doesn't know about.  Also, new parameters
+might be added to existing updates.  It is safe to ignore these as well, and
+just use the parameters known when the client was implemented.
```

Comment:
It is very obvious once you ask yourself the question "hhm, can new events be added or not?", but it is easy to write code like
`getattr(self, "_handle_"+event)(*args)`, see it working well, and not think about what happens in later versions.